subworkflows/local/bam_variant_calling_germline_all/main.nf

```diff
@@ -232,5 +241,9 @@ workflow BAM_VARIANT_CALLING_GERMLINE_ALL {
     strelka_vcf
     tiddit_vcf
 
+    haplotypecaller_vcf_tbi
+    manta_vcf_tbi
+    strelka_vcf_tbi
```

Comment:
What's happening there?

Reply:
nevermind, just realised it was the `emit:`

Reply:
I'm emitting the tbi-files for the vcf-files. The tbi-files are needed for running `bcftools concat` on the coresponding vcf-files.